e2e/specs/st_tooltips.spec.js

```diff
@@ -22,8 +22,8 @@ ante accumsan, a egestas urna aliquet. Nullam eget\nsapien eget diam euismod e\
 leifend. Nulla purus enim, finibus ut velit eu,\nmalesuada dictum nulla. In no\
 n arcu et risus maximus fermentum eget nec ante.`;
 
-const tooltipCodeBlock1 = `This\nis\na\ncode\nblock!\n`;
-const tooltipCodeBlock2 = `for i in range(10):\n    x = i * 10\n    print(x)\n`;
+const tooltipCodeBlock1 = `This\nis\na\ncode\nblock!`;
+const tooltipCodeBlock2 = `for i in range(10):\n    x = i * 10\n    print(x)`;
```

Comment:
Is this an expected change? Why do we need to remove the newlines? (Maybe they should never have been there?)

Reply:
I'm sorry to admit Karen and I dug for a couple hours and couldn't come up with a good explanation as to why these new lines break Cypress tests after updating the `react-syntax-highlighter` dependency 😓